library/src/main/java/com/mapzen/valhalla/ValhallaRouter.kt

```diff
@@ -95,6 +96,7 @@ open class ValhallaRouter : Router, Runnable {
 
     override fun run() {
         var jsonString = gson.toJson(getJSONRequest()).toString()
+        Log.v("a",jsonString)
```

Comment:
I've removed it, it was just a log I used for debugging which I forgot to remove :)